test/regress/regress-2136.txt

```diff
@@ -0,0 +1,11 @@
+;;; TOOL: run-spec-wasm2c
+;;; ARGS: --debug-names
+(module
+ (func $B0 (export "$B0") (param $B0 i32)
```

Comment:
I'm not huge fan of these tests call "regress-XXX" .. can you name the test something like `test/wasm2c/unique-names.txt` or something like that?

Or do you refer this style?

Reply:
Sure, happy to. I don't really have a preference; was just trying to match past practice.

Reply:
Renamed to test/wasm2c/duplicate-names.txt